### ISSUE TITLE
Windows - Set Tesseract path

### ISSUE THREAD

**Issue by davidb1** [sirfz/tesserocr#232]
Is it possible to pass a tesseract path to tesserocr? I don't see a way to set it and `LD_LIBRARY_PATH` doesn't seem to do anything. tesserocr keeps using ver `4.00` when I have `5 alpha` installed

**Comment by sirfz** [comment]
You can pass the `path` parameter to the `PyTessBaseAPI` initializer. Check the `__cinit__` (or `InitFull`) method for the full list of parameters.

**Comment by sirfz** [comment]
Unless you mean tesseract itself and not the language files. The binaries are linked during compilation time.

**Comment by davidb1** [comment]
Yup meant tesseract itself. Any chance for a build with `5 alpha`?

**Comment by sirfz** [comment]
You can try building it and open an issue if it fails.

**Comment by vladvrabie** [comment]
I tried and it didn't work for me;
I have 2 versions of tesseract installed - one through the guide here with conda - another from uni-mannheim.de
I tried this code:
`tesseract = tesserocr.PyTessBaseAPI()`
`print("Default tesseract version: ", tesseract.Version())`

`tesseract.Init(path='C:\\Program Files\\Tesseract-OCR\\', lang='deu+eng')`
`# File "tesserocr.pyx", line 1202, in tesserocr.PyTessBaseAPI._init_api`
`# RuntimeError: Failed to init API, possibly an invalid tessdata path: C:\Program Files\Tesseract-OCR\`
`# don't know why`
`print("My tesseract version: ", tesseract.Version())`

The first print works fine. It prints version 4.1.1 as installed by conda, but the `Init` call fails. I gave it the path where my other version is installed (which is tesseract v5.0.0-alpha.20200328).

Also tried without slash at the end, same error. Also tried putting the paths in quotes (`'"path"'`), same error.
